src/EnergyPlus/FluidProperties.cc

```diff
@@ -4899,6 +4899,7 @@ namespace FluidProperties {
 		int GlycolNum;
 		bool LowErrorThisTime;
 		bool HighErrorThisTime;
+		static bool showedInterpError (false);
```

Comment:
@JasonGlazer Please don't add new function static variables. It just makes unit testing more difficult and require effort to put it in an anonymous namespace and clear_state in the future. Just go ahead and put this new variable into an anonymous namespace and clear_state now.